README-mac.md

```diff
@@ -82,6 +82,8 @@ make xcode/test-debug
 
 ## Try it in the console
 
+Must move to the working directory with config files before using **rime_api_console**.
```

Comment:
What's "the working directory with config files"?
The program can be run from any where, not limited to a particular directory. Of course you can do this too:
This sentence doesn't clarify the correct usage better than the commands that follow.